astropy/utils/console.py

```diff
@@ -631,10 +637,13 @@ def _update_ipython_widget(self, value=None):
         # if none exists.
         if not hasattr(self, '_widget'):
             # Import only if an IPython widget, i.e., widget in iPython NB
-            from IPython.html import widgets
+            try:
+                from ipywidgets import widgets
+            except ImportError:
+                from IPython.html import widgets
             from IPython.display import display
 
-            self._widget = widgets.FloatProgressWidget()
+            self._widget = widgets.FloatProgress()
```

Comment:
Will `FloatProgress` work (as opposed to `FloatProgressWidget`) in all versions, or should that be version dependent too?